src/utils/hooks/useUpdateReviewResponses.ts

```diff
@@ -2,31 +2,55 @@ import {
   ReviewResponse,
   ReviewResponseDecision,
   ReviewResponseRecommendedApplicantVisibility,
+  useUpdateReviewMutation,
   useUpdateReviewResponseMutation,
 } from '../generated/graphql'
 
 // below lines are used to get return type of the function that is returned by useUpdateReviewResponseMutation
 type UseUpdateReviewMutationReturnType = ReturnType<typeof useUpdateReviewResponseMutation>
 type PromiseReturnType = ReturnType<UseUpdateReviewMutationReturnType[0]>
 
-// hook used to update review response, comment and decision, use as per type definition below
-// also computes and updates recommendedApplicantVisibility
-type UseUpdateReviewResponse = () => (
-  reviewResponse: ReviewResponse,
-) => PromiseReturnType
+// Hook to update review responses. Either updates individual response or a
+// batch via updating review object.
+// Also computes recommendedApplicantVisibility
+type UseUpdateReviewResponse = () => {
+  updateReviewResponse: (reviewResponse: ReviewResponse) => PromiseReturnType
+  updateMultipleReviewResponses: any
+}
 
-const useUpdateReviewResponse: UseUpdateReviewResponse = () => {
-  const [updateReviewResponse] = useUpdateReviewResponseMutation()
+const useUpdateReviewResponses: UseUpdateReviewResponse = () => {
+  const [updateReviewResponseMutation] = useUpdateReviewResponseMutation()
+  const [updateMultipleResponsesMutation] = useUpdateReviewMutation()
 
-  return async (reviewResponse) =>
-    updateReviewResponse({
+  const updateReviewResponse = async (reviewResponse: ReviewResponse) =>
+    updateReviewResponseMutation({
       variables: {
         id: reviewResponse.id,
         comment: reviewResponse.comment,
         decision: reviewResponse.decision,
         recommendedApplicantVisibility: computeVisibility(reviewResponse),
       },
     })
+
+  const updateMultipleReviewResponses = async (responses: ReviewResponse[]) => {
+    const reviewId = responses[0].review?.id
+    if (!reviewId) return
+    const reviewPatch = {
+      reviewResponsesUsingId: {
+        updateById: responses.map((response) => ({
+          id: response.id,
+          patch: {
+            comment: response.comment,
+            decision: response.decision,
+            recommendedApplicantVisibility: computeVisibility(response),
+          },
+        })),
+      },
+    }
+    updateMultipleResponsesMutation({ variables: { reviewId, reviewPatch } })
+  }
+
+  return { updateReviewResponse, updateMultipleReviewResponses }
```

Comment:
Hook now returns two methods instead of just a single one for single response update.